src/zeit/content/volume/browser/reference.py

```diff
@@ -0,0 +1,66 @@
+from zeit.cms.i18n import MessageFactory as _
+from zope.browserpage import ViewPageTemplateFile
+import pkg_resources
+import zeit.cms.browser.objectdetails
+import zeit.cms.browser.view
+import zeit.cms.repository.interfaces
+import zeit.content.volume.interfaces
+import zeit.edit.browser.form
+import zope.component
+import zope.formlib.form
+import zope.formlib.interfaces
+
+
+class EditReference(zeit.edit.browser.form.InlineForm):
+
+    legend = ''
+    undo_description = _('edit volume teaser text')
+
+    form_fields = zope.formlib.form.FormFields(
+        zeit.content.volume.interfaces.IVolumeTeaserTextReference,
+        # support read-only mode, see
+        # zeit.content.article.edit.browser.form.FormFields
+        render_context=zope.formlib.interfaces.DISPLAY_UNWRITEABLE).select(
+        'teaser_text')
+
+    @property
+    def prefix(self):
+        return 'reference-details-%s' % self.context.target.uniqueId
+
+
+class ReferenceDetailsHeading(zeit.cms.browser.objectdetails.Details):
+
+    template = ViewPageTemplateFile(pkg_resources.resource_filename(
+        'zeit.cms.browser', 'object-details-heading.pt'))
+
+    def __init__(self, context, request):
+        super(ReferenceDetailsHeading, self).__init__(context.target, request)
+
+    def __call__(self):
+        return self.template()
+
+
+class Display(zeit.cms.browser.view.Base):
+
+    def description(self):
+        volume = self.context.target
+        return "{}, Jahrgang: {}, Ausgabe {}".format(
+            volume.product.title,
+            volume.year,
+            volume.volume,
+        )
+
+    def cover_image(self):
+        height = 300
+
+        if not self.context.target.covers['printcover']:
+            return ''
+        repository = zope.component.getUtility(
+            zeit.cms.repository.interfaces.IRepository)
+        cover_url = '{}{}/@@raw'.format(
+            self.url(repository),
+            self.context.target.covers['printcover'].variant_url(
+                'original', thumbnail=True)
```

Comment:
Hatte gedacht in `z.c.cp` hatten wir das schonmal einfacher, dort steht aber auch `return '%s%s/@@raw' % (self.url(repository), image.variant_url(image_pattern, thumbnail=True))`. :trollface: 

Dann bleibt aber immer noch offen, dass auch ein Einzelbild referenziert sein kann. Die `imageGroupSource` akzeptiert aus historischen Gründen leider beides. Vielleicht eigene Source die nur Bildergruppen akzeptiert?